docs/en/writing-running-appium/android/android-screen-streaming.md

```diff
@@ -50,6 +50,16 @@ driver.execute_script('mobile: startScreenStreaming', {
 })
 ```
 
+```javascript
+// Javascript
+driver.execute('mobile: startScreenStreaming', {
+    'width': 1080,
```

Comment:
I assume in javascript quotes around key names are redundant unless they contain any special characters